### PR TITLE
style: logo has margin y:0.75rem on mobile

### PR DESCRIPTION
**Pull request by hetd54**
NC before and after:

nc-before
nc-after

Example Site before and after:

margin-before
margin-after

### CODE REVIEW

File: packages/gatsby-theme-project-portal/src/components/Navbar.tsx
Comment (hollandjg) [inline packages/gatsby-theme-project-portal/src/components/Navbar.tsx:80]: to keep the old behavior for large screens, you could add 
```suggestion
              className="block mx-4 my-3 xl:my-auto overflow-hidden text-nav text-black font-bold flex gap-4 items-center whitespace-nowrap"
```

Comment (hollandjg) [reply packages/gatsby-theme-project-portal/src/components/Navbar.tsx:80]: You might also need to edit line 88 to keep the position of the menu below the top bar:

Currently:
Screenshot 2023-08-01 at 07 16 52


With 5.5-rem set on line 88
```js
 <div className={"absolute top-[5.5rem]...
```
Screenshot 2023-08-01 at 07 17 07

**Review by hollandjg — CHANGES_REQUESTED** [review]
Hey Heather, thanks for looking at this! I've got a couple of tailwind suggestions to only change the spacing on mobile and keep the rest of the design the same.

**Review by jashlu — APPROVED** [review]
Look's great thanks!

**Review by hollandjg — APPROVED** [review]
Hey Heather, this looks great! It's blocked by #561 right now, so as soon as that is merged, this will be good to go. Thanks!

**Pull request merged**
